Cargo.toml

```diff
@@ -15,6 +15,7 @@ members = [
     "capnp-rpc/examples/calculator",
     "capnp-rpc/examples/pubsub",
     "capnp-rpc/test",
+    "capnp-testdata",
```

Comment:
This is more of an example than a test, right? I think it would be more appropriate to put it under `capnp-rpc/examples` or in the toplevel `example` directory.